src/hotspot/cpu/ppc/vm_version_ppc.cpp

```diff
@@ -109,27 +95,10 @@ void VM_Version::initialize() {
     FLAG_SET_ERGO(TrapBasedRangeChecks, false);
   }
 
-  // Power7 and later.
-  if (PowerArchitecturePPC64 > 6) {
-    if (FLAG_IS_DEFAULT(UsePopCountInstruction)) {
-      FLAG_SET_ERGO(UsePopCountInstruction, true);
-    }
-  }
-
-  if (!VM_Version::has_isel() && FLAG_IS_DEFAULT(ConditionalMoveLimit)) {
+  if (FLAG_IS_DEFAULT(ConditionalMoveLimit)) {
```

Comment:
This is incorrectly replaced. Should get removed. (Causes the mentioned test failures.)